### ISSUE TITLE
Registration through API Only

### ISSUE THREAD

**Issue by mprajescu** [Tzahi12345/YoutubeDL-Material#700]
Is there a way to enable registration through API only and disable it through the GUI? 
Also, there is one more thing. When registration through the GUI is disabled, the LOGIN TAB is still showing half-way through. 
Is it possible to take the whole width? 

![image](https://user-images.githubusercontent.com/18659386/177532943-034688d4-38e2-4fb1-807c-99c17a58853c.png)

**Comment by Tzahi12345** [comment]
No, that isn't possible. Is there a specific reason you want it to be API only?

Just put up a PR with the UI fixes, will probably update it later to add some more improvements: https://github.com/Tzahi12345/YoutubeDL-Material/pull/702
